addons/packages/antrea/1.2.3/bundle/config/values.yaml

```diff
@@ -3,6 +3,7 @@
 
 ---
 infraProvider: vsphere
+nodeSelectors: null
```

Comment:
nit: Please change to nodeSelector everywhere to match k8s deployment key.

Reply:
Updated